src/routes/RequestsRoute.js

```diff
@@ -150,6 +150,11 @@ class RequestsRoute extends React.Component {
         path: 'manualblocks/%{activeRecord.blockId}',
       },
     },
+    automatedPatronBlocks: {
+      type: 'okapi',
+      records: 'automatedPatronBlocks',
+      path: 'automated-patron-blocks/%{activeRecord.patronId}',
+    },
```

Comment:
One patron can have above 100 automated blocks. Probably `limit` should be added here

Reply:
The story has no requirements about restrictions, I think we will be able to add limit later in case of necessarily.